drexelundweiss/__init__.py

```diff
@@ -18,74 +18,90 @@
 # You should have received a copy of the GNU General Public License
 # along with SmartHomeNG. If not, see <http://www.gnu.org/licenses/>.
 #
-# encoding=utf8  
+# encoding=utf8
 
 import logging
 import threading
-import serial
 import os.path
 import time
 import string
 import re
 from lib.model.smartplugin import SmartPlugin
+from bin.smarthome import VERSION
+
+try:
+    import serial
+    REQUIRED_PACKAGE_IMPORTED = True
+except Exception:
+    REQUIRED_PACKAGE_IMPORTED = False
```

Comment:
This could also be solved using a more general implementation.

I don't like general stuff implemented in several plugins.

Maybe the import can be done by a `Plugins. import ()` which returns the imported module or None in case of an Error. It could also log a general message.